### DISCUSSION TITLE
db.null to replace column

### DISCUSSION THREAD

**Discussion by johaven** [sqlalchemy/sqlalchemy#6976] [Usage Questions] [answered]
With 13.x it was possible to use null values to replace empty columns:
```python
qg = (db.session.query(User, *[db.null()] * len(User.__table__.columns), ...)
```
```bash
All selectables passed to CompoundSelect must have identical numbers of columns; select #1 has 30 columns, select #2 has 11
```

Is there an alternative solution for 14.x ?

I read the changelog it seems to be wanted but it's annoying ...

**Comment by zzzeek** [comment]
hi -

it looks like you are creating a union somewhere even though this is not shown in your line above.  Nothing has changed as far as the public facing version of this API.  for further assistance, please supply adequate detail what you are doing, thanks.

**Comment by zzzeek** [comment, marked as answer]
sorry, this is not expected, will open a bug.
